src/core/Alert/index.tsx

```diff
@@ -1,14 +1,8 @@
-import { StylesProvider } from "@material-ui/core/styles";
 import { Alert as RawAlert, AlertProps } from "@material-ui/lab";
 import React from "react";
 
 const Alert = (props: AlertProps): JSX.Element => {
-  return (
-    <StylesProvider injectFirst>
```

Comment:
we now have `injectFirst` in storybook config and in the product project (Aspen), so no need to `inejctFirst` in components anymore!